File: dofun/safe/deviceinfo/info/DebugInfo.java
package com.dofun.safe.deviceinfo.info;

import android.content.Context;
import androidx.core.util.Pair;
import com.dofun.safe.deviceinfo.utils.CommandUtils;
import com.dofun.safe.deviceinfo.utils.DebugUtils;
import java.util.ArrayList;
import java.util.List;
import org.json.JSONObject;

/* loaded from: classes8.dex */
public class DebugInfo {
    public static JSONObject getDebugInfoJson(Context context) {
        return CommandUtils.convertToJson(getDebugInfo(context));
    }

    public static List<Pair<String, String>> getDebugInfo(Context context) {
        List<Pair<String, String>> list = new ArrayList<>();
        list.add(new Pair<>("debugOpen", DebugUtils.isOpenDebug(context) + ""));
        list.add(new Pair<>("usbDebugStatus", DebugUtils.getUsbDebugStatus()));
        list.add(new Pair<>("debugVersion", DebugUtils.isDebugVersion(context) + ""));
        list.add(new Pair<>("debugConnected", DebugUtils.isDebugConnected() + ""));
        return list;
    }
}

File: app/src/main/java/com/dofun/safe/deviceinfo/info/DebugInfo.kt
package com.dofun.safe.deviceinfo.info

import android.content.Context
import androidx.core.util.Pair
import com.dofun.safe.deviceinfo.utils.CommandUtils
import com.dofun.safe.deviceinfo.utils.Constants
import com.dofun.safe.deviceinfo.utils.DebugUtils
import org.json.JSONObject

object DebugInfo {
    @JvmStatic
    fun getDebugInfoJson(context: Context): JSONObject {
        return CommandUtils.convertToJson(getDebugInfo(context))
    }

    @JvmStatic
    fun getDebugInfo(context: Context): List<Pair<String, String>> {
        val list = ArrayList<Pair<String, String>>()
        list.add(Pair("debugOpen", DebugUtils.isOpenDebug(context).toString()))
        list.add(Pair("usbDebugStatus", DebugUtils.getUsbDebugStatus() ?: Constants.UNKNOWN))
        list.add(Pair("debugVersion", DebugUtils.isDebugVersion(context).toString()))
        list.add(Pair("debugConnected", DebugUtils.isDebugConnected().toString()))
        return list
    }
}
